templates/403.html

```diff
@@ -0,0 +1,9 @@
+{% extends "base.html" %}
```

Comment:
can we make the rate limiting return http 429 instead of 403? (https://django-ratelimit.readthedocs.io/en/latest/cookbook/429.html)